### PR TITLE
utility function

### PR DESCRIPTION
**Pull request by izulin**
### Context

Utility function for filling range with source values from other range.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1.
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation,
- [x] I described the modification in the CHANGELOG.md file.

### CODE REVIEW

File: test/engine.spec.ts [hunk 1]
```diff
@@ -773,3 +774,14 @@ describe('Graph dependency topological ordering module', () => {
...
 })
+
+describe('#getFillRangeData', () => {
+  it('should properly apply wrap-around', () => {
+    const engine = HyperFormula.buildFromArray([[1, '=A1'], ['=$A$1', '2']])
+
+    expect(engine.getFillRangeData(
+      AbsoluteCellRange.fromCoordinates(0, 0, 0, 1, 1),
+      AbsoluteCellRange.fromCoordinates(0, 1, 1, 3, 3))
+    ).toEqual([['2', '=$A$1', '2'], ['=A3', 1, '=C3'], ['2', '=$A$1', '2']])
```
[the remark this answers is not in the document]
Comment (izulin) [reply test/engine.spec.ts:785]: Yes, they may overlap - I think this is what we agreed upon

[the remark this answers is not in the document]
Comment (wojciechczerniak) [reply test/engine.spec.ts:785]: Cool 👍

[the remark this answers is not in the document]
Comment (izulin) [reply test/engine.spec.ts:785]: ok, I'll add some tests soon

**Review by wojciechczerniak — APPROVED** [review]
Thanks for addressing all my comments! 🏆

**Pull request merged**